app.py

```diff
@@ -29,14 +29,16 @@
     }
 )
 
+RATELIMIT_STORAGE_URL = os.environ.get("RATELIMIT_STORAGE_URL", "memory://")
```

Comment:
You have a typo here. The variable name is defined somewhere else and it does not end with an L. Please use `RATELIMIT_STORAGE_URI`.

Reply:
By the way. Good choice of default mechanics here.